packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -29,11 +29,13 @@ import '../runner/flutter_command.dart';
 /// in ways that prevent them from ever upgrading Flutter again!
 const Map<String, String> kManuallyPinnedDependencies = <String, String>{
   // Add pinned packages here. Please leave a comment explaining why.
+  'dwds': '22.1.0', // https://github.com/flutter/flutter/issues/140796
```

Comment:
Why does DWDS need to be pinned?

Reply:
Never mind:

```
flutter%20sdk/packages/flutter_tools/lib/src/isolated/devfs_web.dart:102:16: Error: The method 'WebExpressionCompiler.initialize' has fewer positional arguments than those of overridden method 'ExpressionCompiler.initialize'.
  Future<void> initialize({String? moduleFormat, bool? soundNullSafety}) async {}
               ^
/home/chrome-bot/.pub-cache/hosted/pub.dev/dwds-23.0.0/lib/src/services/expression_compiler.dart:86:16: Context: This is the overridden method ('initialize').
  Future<void> initialize(CompilerOptions options);
               ^
flutter%20sdk/packages/flutter_tools/lib/src/isolated/devfs_web.dart:289:23: Error: The argument type 'Uri?' can't be assigned to the parameter type 'BuildSettings'.
 - 'Uri' is from 'dart:core'.
 - 'BuildSettings' is from 'package:dwds/src/loaders/strategy.dart' ('/home/chrome-bot/.pub-cache/hosted/pub.dev/dwds-23.0.0/lib/src/loaders/strategy.dart').
        packageConfig.toPackageUri(
                      ^
```

https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8759996669305826785/+/u/flutter_config_--clear-features/stdout